### PR TITLE
Options for `SectorPreset` overriding plant launch fields.

### PR DESCRIPTION
**Pull request by MEEPofFaith**
In the case that you have something pre-built and you don't want a loadout to destroy that.

https://user-images.githubusercontent.com/54301439/187333583-ab5f69e8-926f-4155-a345-1a78443e5b5d.mp4

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by BlueWolf3682** [comment]
p l a n t

**Comment by MEEPofFaith** [comment]
🍃

File: core/src/mindustry/ui/dialogs/PlanetDialog.java [hunk 1]
```diff
@@ -1160,7 +1160,7 @@ void playSelected(){
                 //free launch.
                 control.playSector(sector);
             }else{
-                CoreBlock block = from.planet.allowLaunchSchematics ? (from.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)from.planet.defaultCore) : (CoreBlock)from.planet.defaultCore;
+                CoreBlock block = sector.allowLaunchSchematics() ? (sector.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)sector.planet.defaultCore) : (CoreBlock)sector.planet.defaultCore;
```
Comment (Anuken) [inline core/src/mindustry/ui/dialogs/PlanetDialog.java:1163]: Why are you changing sector.info.bestCoreType? That looks very dangerous. You should care about the best core from the **launch** sector.

File: core/src/mindustry/ui/dialogs/PlanetDialog.java [hunk 2]
```diff
@@ -1160,7 +1160,7 @@ void playSelected(){
                 //free launch.
                 control.playSector(sector);
             }else{
-                CoreBlock block = from.planet.allowLaunchSchematics ? (from.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)from.planet.defaultCore) : (CoreBlock)from.planet.defaultCore;
+                CoreBlock block = from.allowLaunchSchematics() ? (from.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)from.planet.defaultCore) : (CoreBlock)from.planet.defaultCore;
```
Comment (Anuken) [inline core/src/mindustry/ui/dialogs/PlanetDialog.java:1163]: Okay but now you changed too many things, you're checking if the **from** sector allows launch schematics, which is not what you want, it should still be `sector.allowLaunchSchematics()`.

**Pull request merged**